plutarch-test/common/Plutarch/Test/Run.hs

```diff
@@ -20,11 +27,15 @@ import Test.Hspec.Core.Spec (SpecTree, Tree (Leaf, Node, NodeWithCleanup), runSp
   untracked golden files.
 -}
 noUnusedGoldens :: Spec -> IO ()
-noUnusedGoldens spec = do
+noUnusedGoldens = noUnusedGoldens' def
+
+-- | Like 'noUnusedGoldens' but takes a custom path to the golden storage.
+noUnusedGoldens' :: GoldenConf -> Spec -> IO ()
```

Comment:
How does this function behave when the test spec uses `pgoldenSpec'` multiple times but with _different_ config arguments?

Reply:
It would break. I did test it on another project when the same conf was used and it did work.

Unsure whether there's a solution to the different config issue.